modules/shared-utils/src/cljdoc/util.clj

```diff
@@ -6,7 +6,8 @@
   what's on the classpath."
   (:require [clojure.edn]
             [clojure.java.io :as io]
-            [clojure.string :as string])
+            [clojure.string :as string]
+            [clojure.edn :as edn])
```

Comment:
`clojure.edn` has already been required in it's fully qualified form above.
`clojure.walk` is missing a require. 🙂

Reply:
whoops. missed that.